lib/web_ui/lib/src/ui/painting.dart

```diff
@@ -1455,6 +1455,12 @@ class ImageFilter {
     //    if (matrix4.length != 16)
     //      throw ArgumentError('"matrix4" must have 16 entries.');
   }
+
+  ImageFilter.compose({required ImageFilter outer, required ImageFilter inner}) {
```

Comment:
Does ColorFilter implement ImageFilter in this file?

Reply:
No it doesn't. I'll try adding it to the web engine too.

Reply:
The Web version must be API-compatible with the mobile version, so let's use the same class hierarchy.